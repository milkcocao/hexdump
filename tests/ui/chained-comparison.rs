use anyhow::{ensure, Result};

fn main() -> Result<()> {
    // `ensure!` must not partition this in