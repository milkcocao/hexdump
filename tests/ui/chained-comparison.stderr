error: comparison operators cannot be chained
 --> tests/ui/chained-comparison.rs:6:19
  |
6 |     ensure!(false == false == true);
  |                   ^^    